bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GenericItem.java

```diff
@@ -360,7 +360,7 @@ public void addTags(String... tags) {
 
     @Override
     public void removeTag(String tag) {
-        tags.remove(tag);
+        tags.remove(tags.stream().filter(t -> t.equalsIgnoreCase(tag)).findFirst().orElse(""));
```

Comment:
`""` seems artibrary here. Might `orElse(tag)` be the better choice (would at least result in the identical call as in the old code)?